src/Resources/skeleton/query/QueryHandler.tpl.php

```diff
@@ -0,0 +1,20 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+<?= $use_statements ?>
+
+#[AsMessageHandler]
+class <?= $class_name ?> implements QueryHandler
+{
+    /**
+    */
+    public function __construct()
+    {
+    }
+
+    public function __invoke(<?= $query_class_name ?> $query): Collection
+    {
+        return new Collection([]);
+    }
```

Comment:
**suggestion**: I would not specify a return type (or method body), because it could be anything from a single ID to a whole collection of objects.